types/ckeditor__ckeditor5-core/src/command.d.ts

```diff
@@ -4,15 +4,161 @@ import Editor from './editor/editor';
+     *        cmd.refresh();
+     *      };
+     *
+     *      function forceDisable( evt ) {
+     *        evt.return = false;
+     *        evt.stop();
+     *      }
+     *    }
+     *
+     *    // Usage:
+     *
+     *    // Disabling the command.
+     *    const enableBold = disableCommand( editor.commands.get( 'bold' ) );
+     *
+     *    // Enabling the command again.
+     *    enableBold();
+     */
+    get isEnabled(): boolean;
+    protected set isEnabled(val: boolean);
+
+    /**
+     * A flag indicating whether a command execution changes the editor data or not.
+     *
+     * Commands with `affectsData` set to `false` will not be automatically disabled in
+     * the {@link module:core/editor/editor~Editor#isReadOnly read-only mode} and
+     * {@glink features/read-only#related-features other editor modes} with restricted user write permissions.
+     *
+     * **Note:** You do not have to set it for your every command. It is `true` by default.
+     */
+    readonly affectsData: boolean;
+
+    /**
+     * Refreshes the command. The command should update its {@link #isEnabled} and {@link #value} properties
+     * in this method.
+     *
+     * This method is automatically called when
+     * {@link module:engine/model/document~Document#event:change any changes are applied to the document}.
+     */
     refresh(): void;
+
+    /**
+     * Disables the command.
+     *
+     * Command may be disabled by multiple features or algorithms (at once). When disabling a command, unique id should be passed
+     * (e.g. feature name). The same identifier should be used when {@link #clearForceDisabled enabling back} the command.
+     * The command becomes enabled only after all features {@link #clearForceDisabled enabled it back}.
+     *
+     * Disabling and enabling a command:
+     *
+     *    command.isEnabled; // -> true
+     *    command.forceDisabled( 'MyFeature' );
+     *    command.isEnabled; // -> false
+     *    command.clearForceDisabled( 'MyFeature' );
+     *    command.isEnabled; // -> true
+     *
+     * Command disabled by multiple features:
+     *
+     *    command.forceDisabled( 'MyFeature' );
+     *    command.forceDisabled( 'OtherFeature' );
+     *    command.clearForceDisabled( 'MyFeature' );
+     *    command.isEnabled; // -> false
+     *    command.clearForceDisabled( 'OtherFeature' );
+     *    command.isEnabled; // -> true
+     *
+     * Multiple disabling with the same identifier is redundant:
+     *
+     *    command.forceDisabled( 'MyFeature' );
+     *    command.forceDisabled( 'MyFeature' );
+     *    command.clearForceDisabled( 'MyFeature' );
+     *    command.isEnabled; // -> true
+     *
+     * **Note:** some commands or algorithms may have more complex logic when it comes to enabling or disabling certain commands,
+     * so the command might be still disabled after {@link #clearForceDisabled} was used.
+     */
     forceDisabled(id: string): void;
+
+	/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/ckeditor__ckeditor5-core/src/command.d.ts;
  Line: 138;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.